.rhcicd/clowdapp-engine.yaml

```diff
@@ -248,6 +256,15 @@ parameters:
 - name: OB_TOKEN_CLIENT_SECRET
   description: Client Secret to get a Token from SSO
   value: ob.token.client.secret
+- name: QUARKUS_LOG_CLOUDWATCH_BATCH_PERIOD
+  description: Period between two batch executions. Defaults to 5s.
+  value: 5s
+- name: QUARKUS_LOG_CLOUDWATCH_BATCH_SIZE
+  description: Number of log events sent to CloudWatch per batch. Defaults to 10,000 which is the maximum number of log events per batch allowed by CloudWatch.
+  value: "10000"
+- name: QUARKUS_LOG_CLOUDWATCH_MAX_QUEUE_SIZE
+  description: Optional maximum size of the log events queue. If this is not set, the queue will have a capacity of Integer#MAX_VALUE.
+  value: "100000"
```

Comment:
This is **NOT** the default value in `quarkus-logging-cloudwatch`.

This env var is meant to be used to prevent the memory leak that we detected on prod in July. The underlying (and unfixed) cause of the memory leak is described in https://github.com/aws/aws-sdk-java/issues/2807.

`100000` is a starting point but I'm pretty sure we will lower that value soon.